src/libraries/System.Net.Mail/tests/Unit/MailAddressTests/MailAddressDisplayNameTest.cs

```diff
@@ -21,6 +22,16 @@ public void MailAddress_WithUnicodeDisplayAndMailAddress_ToStringShouldReturnDis
             Assert.Equal(string.Format("\"{0}\" <{1}>", DisplayNameWithUnicode, Address), _mailAddress.ToString());
         }
 
+
+        [Fact]
+        public void MailAddress_WithDoubleQuotesDisplayAndMailAddress_ToStringShouldReturnDisplayNameEscapeSequenceAndAddressInAngleBrackets()
+        {
+            MailAddress _mailAddress = new MailAddress(Address, DisplayNameWithDoubleQuotes);
+            Assert.Equal(DisplayNameWithDoubleQuotes,_mailAddress.DisplayName );
```

Comment:
If we're talking about the whitespaces, there's one missing after ',' and one extra before ')'.

Reply:
ok @ManickaP  i will add that and @wfurt  can you help me with similar kind of test cases exist (path or source link ) so i can check out that and add necessary test for it , as well i seen comment of karelz but not quite sure where i understood it well if you can elaborate a bit that will help. thanks

Reply:
@ManickaP  can you help me with similar kind of test cases exist (path or source link ) so i can check out that and add necessary test for it.

Reply:
@wfurt @ManickaP  you want me to add serializable attribute to mailaddress.cs and add test cases for serialization wrapped with try catch ,is it correct , if not then please guide me so i can complete this pr thanks for all help